x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_partition_field/description.tsx

```diff
@@ -23,7 +23,7 @@ export const Description: FC<Props> = memo(({ children }) => {
       description={
         <FormattedMessage
           id="xpack.ml.newJob.wizard.perPartitionCategorization.enable.description"
-          defaultMessage="If per-partition categorization is enabled then categories are determined independently for each value of the partition field."
+          defaultMessage="If enabled, categories are determined independently for each value of the partition field."
```

Comment:
There are some guidelines related to switch labels in https://elastic.github.io/eui/#/forms/selection-controls/guidelines that might be pertinent here re use of "If enabled...". Maybe this title could just be "Per-partition categorization", the toggle/switch label could just be "on" or "off", and the description could be "Determine categories independently....".

Reply:
Thank you for the suggestion. I updated the PR accordingly. The only part that I haven't addressed is the switch label. I'll do it in a separate PR.

([3307583](https://github.com/elastic/kibana/pull/149407/commits/3307583f8964bf9066984a3f6479cc93972f3115))